src/components/VideoPage.js

```diff
@@ -3,81 +3,174 @@ import YouTube from "react-youtube";
-            },
-          }}
-        />
-      </div>
-      <h2 className="card-head">{video.title}</h2>
-      <div className="flex row gray video-actions ">
-        <div>
-          <span className=" sm">{video.views}</span>
-          <span className=" sm">•{video.uploadDate}</span>
-        </div>
-        <div className="align-items-center row flex ">
-          <button
-            className="align-items-center row flex mg-right-half gray md"
-            style={{
-              color:
-                likes.filter((prev) => prev === video.id).length > 0 && "white",
-            }}
-            onClick={() => dispatch({type: "TOGGLE_LIKE", payload: video})}
-          >
-            <ThumbUpAltIcon />
-            <span>{video.totalLikes}</span>
-          </button>
-          <button
-            className="align-items-center row flex mg-right-half gray md"
-            onClick={() => setShowPlayList(true)}
-          >
-            <PlaylistAddIcon />
-            <span>SAVE</span>
+      {error && (
+        <div className="toast-error mg-bottom-1 errorBox-videopage">
+          {error}
+          <button className="outline-none" onClick={() => setError(null)}>
+            X{" "}
           </button>
-          <VideoPagePlayList
-            showNewPlaylist={showNewPlaylist}
-            setShowNewPlaylist={setShowNewPlaylist}
-            showPlayList={showPlayList}
-            setShowPlayList={setShowPlayList}
-            videoId={newVideoId}
+        </div>
+      )}
+      {video === null && (
+        <div className="videopage-loader">
+          <Loader
+            type="Puff"
+            color="#00BFFF"
+            height={100}
+            width={100}
+            timeout={3000} //3 secs
           />
         </div>
-      </div>
+      )}
+      {video !== null && (
+        <>
+          <div className="video-container">
+            <YouTube
+              videoId={video.videoId}
+              className="responsive-iframe"
+              onReady={addToWatchedVideos}
+              opts={{
+                paddingTop: "0",
+                height: "390",
+                width: "700",
+                playerVars: {
+                  autoplay: 1,
+                },
+              }}
+            />
+          </div>
+          <h2 className="card-head">{video.title}</h2>
+          <div className="flex row gray video-actions ">
+            <div>
+              <span className=" sm">{video.views} views</span>
+              <span className=" sm">•{video.uploadDate.slice(0, 10)}</span>
+            </div>
+            <div className="align-items-center row flex ">
+              <button
+                className="align-items-center row flex mg-right-half gray md"
+                style={{
+                  backgroundColor: "black",
+                  color: "white",
+                }}
+                onClick={() => addToLikedVideos(newVideoId)}
+              >
+                <span>add to liked videos</span>
```

Comment:
There should be some indicator that a video is currently liked. Also provide a little bit of white space between span and thumbUp icon.